tests/test_parser/test_openapi.py

```diff
@@ -114,6 +118,19 @@ def test_from_dict_invalid_version(self, mocker):
 
 
 class TestEndpoint:
+    def make_endpoint(self):
```

Comment:
I feel like it probably makes more sense to either just make this a fixture or take advantage of `setup_method` to set `self.endpoint` to be the default endpoint constructed below.

Reply:
I considered a fixture, but you don't really gain anything here since it's not dependent on other fixtures. It's still going to call it every time, just with the overhead of pytest wrangling. `setup_method` could work, but I still hate it 😂. Totally unclear to most pytest users what that does IMO.